src/rpm/signature.rs

```diff
@@ -1,7 +1,6 @@
+// re-export
```

Comment:
Do we really need this re export any more?

Reply:
```rust
use rpm::signature::crypto::pgp::{Signer,Verifier};
```
does not seem much more ergonomic than
```rust
use rpm::crypto::pgp::{Signer,Verifier};
```

I think the clearest would be 

```rust
use rpm::signature::pgp::{Signer,Verifier};
```
All other paths should be deprecated (if public) and removed before 1.0.0